### PR TITLE
Make Responsive

### PR DESCRIPTION
**Pull request by munjalSaksham71**

### CODE REVIEW

File: src/Screen/VideoListing/VideoListing.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .main_container {
-    margin-left: 15rem;
+    /* margin-left: 15rem; */
     justify-content: center;
```
Comment (tusharkandpal) [inline src/Screen/VideoListing/VideoListing.css:2]: Remove this comment!

File: src/Screen/SignupScreen/SignupScreen.css [hunk 2]
```diff
@@ -15,4 +15,20 @@ button, input {
 .signup-button {
     width: 40%;
     margin-left: -5rem;
+}
+
+
+@media only screen and (max-width: 900px) {
+    input{
```
Comment (tusharkandpal) [inline src/Screen/SignupScreen/SignupScreen.css]: Please avoid direct styles on tag, instead use classes

**Review by tusharkandpal — APPROVED** [review]
Everything looks good!

Please give a brief description in your PR about the changes.

File: src/Screen/HistoryScreen/HistoryScreen.jsx [hunk 3]
```diff
@@ -16,8 +16,8 @@ const HistoryScreen = () => {
 
 
   return (
-    <div className="flex-row">
-      <SideBar />
+    <div>
+      <SideBar pageWrapId={'page-wrap'} outerContainerId={'outer-container'} />
```
Comment (tushartiwari7) [inline src/Screen/HistoryScreen/HistoryScreen.jsx:20]: Correct me if I am wrong but You are not using these props in sidebar component so why passing it here?

**Review by tushartiwari7 — APPROVED** [review]
everything looks neat and clean.
Remove console logs.
This PR is good to merge.

File: src/Screen/HistoryScreen/HistoryScreen.jsx (hunk 3 above)
Comment (munjalSaksham71) [reply src/Screen/HistoryScreen/HistoryScreen.jsx:20]: These Props are from ```react-burger-menu``` package. To get Wrap the side bar - So, what pageWrapId is doing here is: ```pageWrapID``` is wrapping the rest of the content on page.

**Pull request merged**